tests/ballerina-compiler-api-test/src/test/resources/test-src/query-expression/query_expression_tests.bal

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we organize these test files in the same directory structure as well? i.e., `test-src/expressions/query_expression_tests.bal`

Reply:
Updated in 011f37a60e1ca34dccec4e3f2700ffe19ab92cd9